src/VirtoCommerce.ElasticAppSearch.Core/Models/Api/Search/SearchQuerySortField.cs

```diff
@@ -0,0 +1,10 @@
+namespace VirtoCommerce.ElasticAppSearch.Core.Models.Api.Search
+{
+    public class SearchQuerySortField
+    {
+        public string Field { get; set; }
+
+        // asc, desc
+        public string Order { get; set; }
```

Comment:
May be introduce `enum`. It will be successfully converted to camelCase if Elastic App Search is case-sensitive due to JSON serialization settings of `ApiClient`: https://github.com/VirtoCommerce/vc-module-elastic-app-search/blob/ee2785689a916036f0fb33d718a0668cf3ac9d77/src/VirtoCommerce.ElasticAppSearch.Data/Services/ApiClient.cs#L27